fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexer.java

```diff
@@ -1807,6 +1868,28 @@ public Builder setLimit(int limit) {
             return this;
         }
 
+        /**
+         * Get the approximate maximum transaction write size. Note that the actual size might be "a little" bigger.
+         * @return the requested write size. Index iteration stops when exceeding this size.
```

Comment:
In general, we try and keep `@return` statements as a single "statement" (without a period), so something like `@return the requested write size`, and then additional clarifying comments go in the body.

Also, looking at "Index iteration stops when exceeding this size.", the way I read that is that it stops, like, "iterating over the index and the build stops" or something when it hits the limit, which I don't think is what you're getting at. I think something like "transactions started as part of the index build will be committed after they exceed this size, and a new transaction will be started", or something like that.